### ISSUE TITLE
Realsense D435I did not turn off after stopping the launch through "Ctrl-C" (IR projector continued to light) on NUC

### ISSUE THREAD

**Issue by amburkoff** [realsenseai/realsense-ros#1237]
<table>
<tr><th>Computer</th><th>NUC8i7BEKQA2</th></tr>
<tr><th>Camera</th><th>D435i</th></tr>
<tr><th>Firmware</th><th>05.12.03</th></tr> 
<tr><th>Cable use</th><th>Original cabel</th></tr>
<tr><th>OS</th><th>Ubuntu 18.04.4</th></tr>
<tr><th>Kernel</th><th>5.3.0-59-generic</th></tr>
<tr><th>ROS Version</th><th>Melodic</th></tr>
<tr><th>LibRealSense</th><th>v2.33.1 (install from apt)</th></tr>
<tr><th>RealSense ROS</th><th>v2.2.13 (install from apt)</th></tr>
</table>

We conducted camera fault tolerance tests. 
The first test was long time test ( ~ 4.5 hours ). During this time camera worked fine.
After that, we immediately conducted short tests (~ one minute each). On the twelfth test, the camera did not turn off after stopping the launch through "Ctrl-C" (*IR projector continued to light*). 
Then we tried to conduct another short test, but the realsense node could not connect to the camera, with the following error:
```
Tue 2020-06-09 16:15:06.919: realsense2_camera_manager:  09/06 16:15:06,919 WARNING [139837322102528] (types.cpp:49) set_xu(ctrl=1) failed! Last Error: Resource temporarily unavailable
Tue 2020-06-09 16:15:06.950: realsense2_camera_manager:  09/06 16:15:06,950 WARNING [139837322102528] (rs.cpp:283) null pointer passed for argument "device"
Tue 2020-06-09 16:15:06.950: realsense2_camera_manager:  [33m[RealSenseNodeFactory::getDevice]: Device 1/1 failed with exception: set_xu(ctrl=1) failed! Last Error: Resource temporarily unavailable [0m
Tue 2020-06-09 16:15:06.950: realsense2_camera_manager:  [31m[RealSenseNodeFactory::getDevice]: The requested device with serial number 846112070905 is NOT found. Will Try again. [0m
Tue 2020-06-09 16:15:12.958: realsense2_camera_manager:  [0m[RealSenseNodeFactory::getDevice]:   [0m
```
We tried to reconnect the camera in a loop.
The launch command is:
```
mon launch realsense2_camera rs_camera.launch serial_no_camera:=<SERIAL_NUMBER>
```
We used `mon launch` to conveniently save logs.
We was able to reconnect camera **only after switching the wire**.
Log files:
- log_09_06_2020-16_15.txt - last fine short test (after this one, camera didn't turn off)
- log_09_06_2020-16_15.txt - camera didn't connect
- log_09_06_2020-16_19.txt - camera didn't connect
- log_09_06_2020-16_20.txt - camera worked fine after swithching wire port

Screenshot reset_usb.py list - https://drive.google.com/file/d/1mn8f5q7XGSVpOa_jlWrhIHW2x8SkZziW/view
log_09_06_2020-16_14.txt - https://drive.google.com/file/d/11MH6N0JiiWIs_KiBZWuP9TXquV7RJVy0/view?usp=sharing
log_09_06_2020-16_15.txt - https://drive.google.com/file/d/1HvKyBXkIrHp0JlkjBz3yC27OsbUGIyrL/view?usp=sharing
log_09_06_2020-16_19.txt - https://drive.google.com/file/d/1fhpC3QHuTLP-CooWnM0NVeQQdlMVPUGp/view?usp=sharing
log_09_06_2020-16_19.txt - https://drive.google.com/file/d/1K0Og31n4MfVdEAgeWTQi7FwEmbJxcOV_/view?usp=sharing

**Comment by doronhi** [comment]
I fixed a stopping issue in version 2.2.14. Could you check it out? I don't think that running the camera for 4.5 hours prior to the start-stop test is needed. It could save you some time.

**Comment by RealSenseSupport** [comment]
Closing this ticket as 2.2.14 has provided the fix. 
Thank you.
